main/seccomp.c

```diff
@@ -41,7 +41,9 @@ int installSyscallFilter (void)
 	seccomp_rule_add (ctx, SCMP_ACT_ALLOW, SCMP_SYS (exit_group), 0);
 
 	// The bowels of stdio want to know the size of a file, even for stdout.
+#if (defined(__SNR_fstat) && __SNR_fstat)
```

Comment:
Just a question.

Is
```
#ifdef __SNR_fstat
```
not enough?

Reply:
Simply defining __SNR_fstat is not enough, because there are architectures that lack __NR_fstat and __PNR_fstat, where __SNR_fstat is an empty macro definition, and therefore results in the error

Reply:
Should we do for __SNR_newfstatat the same as you did for __SNR_fstat?

Reply:
I think it's needed.

Reply:
Thank you. I will update the other #ifdefs.